packages/fx-core/src/component/driver/interface/buildAndDeployArgs.ts

```diff
@@ -58,4 +58,17 @@ export type AxiosHeaderWithLocation = {
   };
 };
 
+export type DeployResult = {
+  id: string;
+  status: number;
+  start_time: string;
+  end_time: string;
+  complete: boolean;
+  active: boolean;
+  is_readonly: boolean;
+  url: string;
```

Comment:
Are these properties all necessary for BI? And I wonder `url` may contains PII. For example, the app service name.

Reply:
I want the URL to be a unique ID for the deployment, and if it is PII data, I can do a hash on it first.